src/encoded/static/components/browse/components/FileEntryBlock.js

```diff
@@ -208,9 +208,10 @@ export class FileEntryBlock extends React.PureComponent {
 
 
     renderName(){
-        const { file, colWidthStyles, label, excludeCheckbox, selectedFiles } = this.props;
+        const { file,  columnHeaders, colWidthStyles, label, excludeCheckbox, selectedFiles } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['label' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4901542)